### DISCUSSION TITLE
Slot support for ui.chat_message

### DISCUSSION THREAD

**Discussion by jzluo** [zauberzeug/nicegui#1772] [Ideas / Feature Requests]
Quasar can do it: https://quasar.dev/vue-components/chat#slots but 

```py
with ui.chat_message('Hello NiceGUI!',):
    ui.button('test')
```

does not work.

**Comment by falkoschindler** [comment]
That's a great idea, @jzluo!
I thought I could quickly add slots to chat_message.js, but unfortunately this seems to break existing functionality and I have no clue why. See draft PR #1774.
Maybe someone can help us out fixing the implementation?

**Comment by alikalik9** [comment]
+1 for this Feature Request :). I am building a llm-chat-application with nicegui (will probably make the repo public once it's ready) and the ui.chat_message is sadly lacking functionality. Adding things like a markdown or copy button would be amazing.

**Comment by falkoschindler** [comment]
I just noticed that this feature has already been released in version 1.4.3. I'll close this discussion.
